src/NCalc/NCalc.csproj

```diff
@@ -22,6 +22,7 @@
     <GenerateNeutralResourcesLanguageAttribute>false</GenerateNeutralResourcesLanguageAttribute>
     <GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
     <GenerateAssemblyFileVersionAttribute>false</GenerateAssemblyFileVersionAttribute>
+    <PackageVersion>3.15.0</PackageVersion>

```

Comment:
One question: can we bump the `Version` instead of introducing `PackageVersion`? what are their differences?